docs/onboarding/PowerAuth-Enrollment-Onboarding-Server-1.5.0.md

```diff
@@ -75,6 +75,53 @@ ALTER TABLE es_onboarding_process
 ```
 
 
+### SCA Result
+
+A new table `es_sca_result` has been created.
+
+
+#### PostgreSQL
+
+```sql
+CREATE SEQUENCE es_sca_result_seq INCREMENT BY 50 START WITH 1 CACHE 2;
```

Comment:
What is the point of caching 2 sequence IDs?

Reply:
Let's drop the explicit CACHE size and use the default.